examples/openai-gpt.ipynb

```diff
@@ -102,12 +103,35 @@
    },
    "outputs": [],
    "source": [
-    "message = ({\"role\": \"user\", \"content\": \"Hello\"},)\n",
-    "openai.chat.completions.create(\n",
+    "message = ({\"role\": \"user\", \"content\": \"Write a 12 word poem about secret agents.\"},)\n",
+    "res = openai.chat.completions.create(\n",
     "    model=\"gpt-3.5-turbo\", messages=message, temperature=0.5, stream=True\n",
     ")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "4a231440",
+   "metadata": {},
+   "source": [
+    "We then use this result in our code to have it logged to AgentOps."
```

Comment:
Awkward wording. I'd write: `"Streamed completions are also automatically logged to AgentOps"`